src/he/mss.h

```diff
@@ -0,0 +1,28 @@
+/* *
+ * Lightway Core
+ * Copyright (C) 2023 Express VPN International Ltd.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+#ifndef HE_MSS_H
+#define HE_MSS_H
+
+#include "he.h"
+#include "he_internal.h"
+
+void he_clamp_mss(uint8_t *packet, size_t length, uint16_t maximum_mss);
```

Comment:
Since it's a public api, let's add docs to this function here.

Also, minor suggestions:
1. We could have this function return `he_return_code_t` instead of `void`
2. `maximum_mss` could be just `mss`, because `mss` already stands for "Maximum Segment Size"